src/mlnet/CodeGenerator/CodeGenerationHelper.cs

```diff
@@ -185,25 +179,55 @@ public void GenerateCode()
                 switch (taskKind)
                 {
                     case TaskKind.BinaryClassification:
-                        var bestBinaryIteration = binaryExperimentResult.BestRun;
-                        bestPipeline = bestBinaryIteration.Pipeline;
-                        bestModel = bestBinaryIteration.Model;
-                        ConsolePrinter.ExperimentResultsHeader(LogLevel.Info, settings.MlTask, settings.Dataset.Name, columnInformation.LabelColumnName, elapsedTime.ToString("F2"), binaryExperimentResult.RunDetails.Count());
-                        ConsolePrinter.PrintIterationSummary(binaryExperimentResult.RunDetails, new BinaryExperimentSettings().OptimizingMetric, 5);
+                        if (completedBinaryRuns.Count > 0)
```

Comment:
We can produce the VS solution without completing any models.

Reply:
Nope